package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "svg-term",
-  "version": "1.1.7",
+  "version": "1.2.0",
```

Comment:
Bump minor version.
No need to bump major version because the new API is backward compatible with the previous version.
Please publish a new NPM version after merging.